### ISSUE TITLE
Feature Request: Android

### ISSUE THREAD

**Issue by alistairholmes** [emilbaehr/automatic-app-landing-page#1]
Thanks for all the hard work, this is a great project. It would be awesome if there could be a version  of this for android as well. #featurerequest

**Comment by rohanagarwal94** [comment]
An Android Support for same Repo. Please star to support - https://github.com/autoswiperight/autoswiperight.github.io

**Comment by CharlieCortial** [comment]
> An Android Support for same Repo. Please star to support - https://github.com/autoswiperight/autoswiperight.github.io

This is not true at all. There is nothing about Android support in the code of your repo. The requested feature isn't working on your website. Misleading and false.

**Comment by rohanagarwal94** [comment]
Hello @CharlieCortial if you opened the website - https://app.swipetinder.me/ , this is clearly for Android app.

**Comment by CharlieCortial** [comment]
Hello @rohanagarwal94 yes it is an Android app, but this Playstore feature is already part of @emilbaehr work. The requested Android Support is about getting a native Android banner for downloading the app featured on the website. It’s not the case of yours.
And yes, I checked your GitHub repo with a simple differential tool, there is no change that would allow this feature, anyway.

**Comment by rohanagarwal94** [comment]
@CharlieCortial I still honestly cannot get the difference.

**Comment by CharlieCortial** [comment]
@rohanagarwal94 It's about a feature called "Smart App Banner" as seen in [this blog post](https://blog.branch.io/how-to-setup-an-ios-and-android-smart-app-banner-with-deep-linking-and-download-tracking/).
You didn't add this feature, you just used the one already present in @emilbaehr 's work.
I don't know how it can be more clear.
